src/components/Book.js

```diff
@@ -1,29 +1,52 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './Book.css';
 
 const Book = ({
   bookId, title, category, handleRemoveBook,
 }) => (
-  <tr>
-    <td>{bookId}</td>
-    <td>{title}</td>
-    <td>{category}</td>
-    <td>
-      <button
-        onClick={() => {
-          const book = {
-            bookId,
-            title,
-            category,
-          };
-          handleRemoveBook(book);
-        }}
-        type="button"
-      >
-        Remove
-      </button>
-    </td>
-  </tr>
+  <div className="book-container">
+    <div className="book-details">
+      <div>
+        <div>{bookId}</div>
+        <div className="title">{title}</div>
+        <div>{category}</div>
```

Comment:
- [ ] Please display in here the book author, it can be hardcoded if you didn't add that feature to your app or you can also implement a small code to generate random names, whichever you choose will be alright:
![image](https://user-images.githubusercontent.com/38522578/128101068-14266544-a76f-4f28-9314-5fb42cb99fa8.png)

- [ ] Please also make it light blue to match the design :)

Reply:
Resolved